RecoMuon/TrackerSeedGenerator/plugins/TSGForOI.h

```diff
@@ -104,7 +104,9 @@ class TSGForOI : public edm::stream::EDProducer<> {
 	edm::ESHandle<Chi2MeasurementEstimatorBase>   estimator_;
 
 	/// Function to find seeds on a given layer
-	void findSeedsOnLayer(const GeometricSearchDet &layer,
+	void findSeedsOnLayer(
+			      const edm::ESHandle<TrackerTopology> tTopo,
```

Comment:
hi @OlivierBondu - please make this argument type either a 
const edm::ESHandle<TrackerTopology> &tTopo
or a const TrackerTopology *tTopo